CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Added
 
+- `sdk.alerts.update_note` to add or update note regarding an alert.
```

Comment:
Can you end the method name with `()`?
`sdk.alerts.update_note()`

Reply:
Done.